index.d.ts

```diff
@@ -27,14 +27,7 @@ declare module '@primer/components' {
     extends StyledSystem.BordersProps,
       StyledSystem.BoxShadowProps {}
 
-  interface PositionProps
-    extends BaseProps,
-      StyledSystem.PositionProps,
-      StyledSystem.ZIndexProps,
-      StyledSystem.TopProps,
-      StyledSystem.RightProps,
-      StyledSystem.BottomProps,
-      StyledSystem.LeftProps {}
```

Comment:
I removed all of these because they are already included in the definition for `StyledSystem.PositionProps`

Reply:
I removed `BaseProps` because the two components that extend `PositionProps` (Position & Popover) already are including `BaseProps` in their definition